binding.gyp

```diff
@@ -71,6 +71,17 @@
           ],
         }],
       ]
+    },
+    {
+      "target_name": "action_after_build",
```

Comment:
This is [the compile target that `node-pre-gyp`](https://github.com/mapbox/node-pre-gyp#2-add-a-new-target-to-bindinggyp) is relying on.